src/lib/app-fns/tx/clearAdmin.tsx

```diff
@@ -28,21 +27,20 @@ interface ClearAdminTxParams {
 
 export const clearAdminTx = ({
   address,
-  contractAddress,
+  messages,
   fee,
   memo,
   client,
   onTxSucceed,
 }: ClearAdminTxParams): Observable<TxResultRendering> => {
   return pipe(
     sendingTx(fee),
-    postTx<ChangeAdminResult>({
-      postFn: () => client.clearAdmin(address, contractAddress, fee, memo),
+    postTx<DeliverTxResponse>({
+      postFn: () => client.signAndBroadcast(address, messages, fee, memo),
     }),
     ({ value: txInfo }) => {
       onTxSucceed?.();
-      const txFee = txInfo.events.find((e) => e.type === "tx")?.attributes[0]
-        .value;
+      const txFee = findAttribute(txInfo.events, "tx", "fee")?.value;
```

Comment:
I think the "?." that you used, it didn't work as expected because it'll throw the error `Could not find attribute '${attrKey}' in first event of type '${eventType}' in first log.` instead. So, I think we can remove `?`.